core/src/main/java/org/opensearch/sql/utils/DateTimeUtils.java

```diff
@@ -294,4 +305,55 @@ private static String normalizeUnit(String rawUnit) {
       }
     }
   }
+
+  public static RexNode transferCompareForDateRelated(
```

Comment:
please add javadoc for public static methods

Reply:
Done.